### PR TITLE
Trento test 05.190, restarting stopped DB

### PR DESCRIPTION
**Pull request by mpagot**
- Implement new test module and add to the scheduling
- More code to the lib, some lib methods no more needs to be public
- Fix sapconf qesap regression configuration

Related ticket: CFSA-917
Verification run: 
- qesap regression gcp sapconf : http://1c242.qa.suse.de/tests/1469 http://openqaworker15.qa.suse.cz/tests/102153#
- trento http://openqaworker15.qa.suse.cz/tests/103169 http://openqaworker15.qa.suse.cz/tests/103170

### CODE REVIEW

**Review by jankohoutek — APPROVED** [review]
LGTM

File: tests/sles4sap/trento/test_hana_restore_stopped.pm [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        '--name=goofy');    # goofy is the original primary site name, from hana_vars in data/sles4sap/qe_sap_deployment/trento_azure.yaml
+    cluster_hdbadm($primary_host, $cmd);
+
+    # Restart the stopped instance
+    my $prov = get_required_var('PUBLIC_CLOUD_PROVIDER');
+    qesap_ansible_cmd(cmd => "sudo crm resource refresh rsc_SAPHana_HDB_HDB00 $primary_host",
+        provider => $prov,
+        filter => $primary_host);
+
+    cluster_wait_status($primary_host, \&condition);
```
Comment (alvarocarvajald) [inline tests/sles4sap/trento/test_hana_restore_stopped.pm:48]: Why not:
```
cluster_wait_status($primary_host, sub { (($_[0] =~ m/.+DEMOTED.+SOK/) && ($_[1] =~ m/.+PROMOTED.+PRIM/)); });
```
Or:
```
cluster_wait_status($primary_host, sub { ((shift =~ m/.+DEMOTED.+SOK/) && (shift =~ m/.+PROMOTED.+PRIM/)); });
```
We lose a bit of readability by not having the named variables in the condition function, but we gain it by having the condition we're comparing to there in the line.

File: tests/sles4sap/trento/test_hana_stop.pm [hunk 2]
```diff
@@ -14,50 +14,23 @@ use utils 'script_retry';
...
-        }
-        $done = (($status{vmhana01} =~ m/.*UNDEFINED.*SFAIL/) && ($status{vmhana02} =~ m/.*PROMOTED.*PRIM/));
-        record_info("SAPHanaSR-showAttr",
-            join("\n\n", "Output : $show_attr",
-                'status{vmhana01} : ' . $status{vmhana01},
-                'status{vmhana02} : ' . $status{vmhana02},
-                "done : $done"));
-    }
-    die "Timeout waiting for hand over" if !$done;
+    cluster_wait_status($primary_host, \&condition);
```
Comment (alvarocarvajald) [inline tests/sles4sap/trento/test_hana_stop.pm:33]: Same question as above.

**Review by alvarocarvajald — APPROVED** [review]
LGTM. But I left 4 small comments nevertheless.

**Pull request merged**